.github/ISSUE_TEMPLATE/formatting.md

```diff
@@ -18,32 +18,13 @@ BEFORE SUBMITTING AN ISSUE:
     This means we have a very high bar for adding new options.
     Find out more: https://prettier.io/docs/en/option-philosophy.html
 
-Tip! Don't write this stuff manually.
+👉 Still sure you want to submit an issue? 🛑 DON'T USE THIS FORM.
+🤖 Let a program fill the issue template for you!
 
 1. Go to https://prettier.io/playground
 2. Paste your code and set options
 3. Press the "Report issue" button in the lower right
+4. Enter a title and describe the expected behavior
+5. Submit
```

Comment:
I am afraid please will be ignore an issue templat - with simple words, they did not see the comment. From my experience, everything in the comments is ignored by many